x/gov/init.go

```diff
@@ -41,28 +41,8 @@ func (*Initializer) FromGenesis(opts weave.Options, db weave.KVStore) error {
 	if err := opts.ReadOptions("governance", &governance); err != nil {
 		return err
 	}
-	// handle election rules
-	rulesBucket := NewElectionRulesBucket()
-	for i, r := range governance.Rules {
-		rule := ElectionRule{
-			Metadata:          &weave.Metadata{Schema: 1},
-			Admin:             r.Admin,
-			Title:             r.Title,
-			VotingPeriodHours: r.VotingPeriodHours,
-			Threshold:         Fraction{Numerator: r.Threshold.Numerator, Denominator: r.Threshold.Denominator},
-		}
-		if r.Quorum.Numerator != 0 || r.Quorum.Denominator != 0 {
-			rule.Quorum = &Fraction{Numerator: r.Quorum.Numerator, Denominator: r.Quorum.Denominator}
-		}
-		if err := rule.Validate(); err != nil {
-			return errors.Wrapf(err, "electionRule #%d is invalid", i)
-		}
-		if _, err := rulesBucket.Create(db, &rule); err != nil {
-			return err
-		}
-	}
 
-	// handle electorate
+	// handle electorate first, as rules refer to them
```

Comment:
This is a greatly improved comment! :golfing_man: 
From previous **what** it is making **why** and now it is helpful!